Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -262,7 +262,12 @@ open class NavigationMapView: UIView {
             fatalError("Access token was not set.")
         }
         
-        mapView = MapView(with: frame, resourceOptions: ResourceOptions(accessToken: accessToken))
+        let tileStoreURL = FileManager.default.urls(for: .documentDirectory,
```

Comment:
The document directory is exposed to the user in the Files application and backed up to iCloud by default. It would be more appropriate to stick the cache in Caches:

https://github.com/mapbox/mapbox-navigation-ios/blob/68c9ca655e7e1bf81436a4ce1f6f855f2b0cf221/Sources/MapboxCoreNavigation/BundleAdditions.swift#L60-L67

Caches may be wiped if the device runs out of space; the idea is that caches can be recreated easily. If this is a problem, then we can stick the cache in Application Support, but make sure to [exclude the folder from iCloud](https://github.com/mapbox/mapbox-gl-native-ios/blob/ddf8e35970d1a8a46171bd798bcfa8f22d2760e0/platform/darwin/src/MGLOfflineStorage.mm#L297).

Reply:
Replaced current `tileStoreURL` with `Bundle.mapboxNavigation.suggestedTileURL?.path`.